src/main/scala/io/archivesunleashed/matchbox/RemoveHTML.scala

```diff
@@ -17,10 +17,16 @@
 package io.archivesunleashed.matchbox
 
 import java.io.IOException
-
 import org.jsoup.Jsoup
 
+/** Remove HTML markup with JSoup. */
 object RemoveHTML {
+
+  /** Remove HTML markup.
```

Comment:
Use 3rd person (descriptive) not 2nd person (prescriptive). 

http://www.oracle.com/technetwork/java/javase/tech/index-137868.html

(Here and elsewhere)